queries.sql

```diff
@@ -8,3 +8,26 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > '10.5';
 SELECT * FROM animals WHERE neutered = 't';
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN '10.4' AND '17.3';
+BEGIN;
+UPDATE animals SET species = 'unespecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
```

Comment:
- [ ] Add SQL code after this line to verify that changes were made. It can look something like this:
```sql
SELECT species from animals
```